s2.asm

```diff
@@ -350,9 +398,9 @@ GameClrRAM:
 ; loc_394:
 MainGameLoop:
 	move.b	(Game_Mode).w,d0
-	andi.w	#$3C,d0
-	jsr	GameModesArray(pc,d0.w)
-	bra.s	MainGameLoop
+	andi.w	#$3C,d0	; limit Game Mode value to $3C max (remove to add more game modes)
```

Comment:
IIRC, the highest bit is set on the title screen, so removing the instruction outright should cause issues.

Reply:
High bit is set during the titlecard, and cleared after; this code is not reachable with high bit set under normal circumstances. You could get there if the game crashed at the titlecard, though.

A better note would be to change it to $7C to allow more game modes.

Reply:
I'll do that